src/idl_gen_cpp.cpp

```diff
@@ -272,8 +271,8 @@ class CppGenerator : public BaseGenerator {
   }
 
   void GenExtraIncludes() {
-    for (std::size_t i = 0; i < opts_.cpp_includes.size(); ++i) {
-      code_ += "#include \"" + opts_.cpp_includes[i] + "\"";
+    for (const auto & cpp_include : opts_.cpp_includes) {
```

Comment:
const auto & cpp_include => const auto &cpp_include

Reply:
Yeah, our `clang-format` should apply this automatically. See `scripts/clang-format-git.sh`

Reply:
This comment applies to all the changes in this PR.

Reply:
`scripts/clang-format-git.sh` doesn't works.